java/jdbc/ConnectionSamples/ADBQuickStart.java

```diff
@@ -0,0 +1,149 @@
+/* Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+Licensed under the Universal Permissive License v 1.0 
+as shown at http://oss.oracle.com/licenses/upl */
+
+/*
+ DESCRIPTION
+ The code sample demonstrates establishing a connection to Autonomous Database (ATP/ADW) using
+ Oracle JDBC driver and Universal Connection Pool (UCP). It does the following.  
+ 
+ (a) Set the connection factory class name to 
+ oracle.jdbc.pool.OracleDataSource before getting a connection.   
+ (b) Set the connection pool properties(e.g.,minPoolSize, maxPoolSize). 
+ (c) Get the connection and perform some database operations. 
+ For a quick test, the sample retrieves 20 records from the Sales History (SH) schema 
+ that is accessible to any DB users on autonomous Database.  
+ 
+ Step 1: Enter the Database details DB_URL and DB_USER. 
+ You will need to enter the DB_PASSWORD of your Autonomous Database through console
+ while running the sample.  
+ Step 2: Download the latest Oracle JDBC driver(ojdbc8.jar) and UCP (ucp.jar) 
+ along with oraclepki.jar, osdt_core.jar and osdt_cert.jar and add to your classpath.  
+ Refer to https://www.oracle.com/database/technologies/maven-central-guide.html               
+ Step 3: Compile and Run the sample. 
+ 
+ SH Schema: 
+ This sample uses the Sales History (SH) sample schema. SH is a data set suited for 
+ online transaction processing operations. The Star Schema Benchmark (SSB) sample schema 
+ is available for data warehousing operations. Both schemas are available 
+ with your shared ADB instance and do not count towards your storage. 
+ ou can use any ADB user account to access these schemas.
+ 
+ NOTES
+ Use JDK 1.8 and above 
+  
+ MODIFIED    (MM/DD/YY)
+ nbsundar    11/09/2020 - Creation 
+ */
+package com.oracle.jdbctest;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Scanner;
+
+import oracle.ucp.jdbc.PoolDataSourceFactory;
+import oracle.ucp.jdbc.PoolDataSource;
+
+public class ADBQuickStart {
+ 
+  // Make sure to have Oracle JDBC release 18c or higher 
+  // to pass TNS_ADMIN as part of a connection URL.
+  // TNS_ADMIN - Should be the path where the client credentials zip file is downloaded. 
+  // wallet_dbname - This is the TNS Alias that you can get from tnsnames.ora.
+  final static String DB_URL="jdbc:oracle:thin:@wallet_dbname?TNS_ADMIN=/Users/test/wallet_dbname/";
```

Comment:
All these fields are not accessed outside of the `main` method. I'd recommend moving them to variables inside `main`.

Reply:
Done